### PR TITLE
Include Solid-compatible app Criteria

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
Please add items

### CODE REVIEW

**Comment by NSeydoux** [comment]
What about the suggestions in 3645f4495699c108058e00d1ed0989d60191ace7 ?

File: pages/use-solid/apps.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ permalink: /use-solid/apps
 
 These Solid-compatible apps will store your data on [your own Pod]({{site.baseUrl}}/use-solid).
 
+The criteria for an app to be Solid-compatible are: 
+* If identifying users is necessary, they must be able to login using their WebID and pointing to the Identity Provider of their choice
+* Data consumed by the app should be fetched from Solid Pods
```
Comment (RubenVerborgh) [inline pages/use-solid/apps.md:11]: I would remove this requirement, or make it optional; data can come from many places.

File: pages/use-solid/apps.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ permalink: /use-solid/apps
...
+* Data generated by the app should be stored in Solid Pods
+* The interaction between the app, the Identity Provider and the Storage Provider(s) (who host Pods) must be compliant with the Solid spec
```
Comment (RubenVerborgh) [inline pages/use-solid/apps.md:13]: There is no actual interaction with storage providers.

```suggestion
* The interaction between the app, pods, and Identity Provider(s) must be compliant with the Solid specification
```

**Comment by Mitzi-Laszlo** [comment]
Re: "anything that may (possibly indirectly) cause harm to anyone.."  It could be argued that all apps have the potential to cause some sort of harm so it's important to specify what type of harm. 

Re: "anything that would be widely considered as unethical, .. " under which ethical framework? 

Re: "generating/promoting/dissemination of hate speech or encouraging violence towards a person or group...anything technically malicious - data theft... sharing of user's information with third-parties without their consent..." how would this be reviewed? Under the test suite?

**Comment by NSeydoux** [comment]
I added a first draft for exclusion criteria in 658f7ab8c3cc39c628adc119215902186bd11b12. 

 > Re: "anything that may (possibly indirectly) cause harm to anyone.." It could be argued that all apps have the potential to cause some sort of harm so it's important to specify what type of harm.

Do you have a specific idea ?

> Re: "anything that would be widely considered as unethical, .. " under which ethical framework?
Re: "generating/promoting/dissemination of hate speech or encouraging violence towards a person or group...anything technically malicious - data theft... sharing of user's information with third-parties without their consent..." how would this be reviewed? Under the test suite?

I guess these can be left to the discretion of the admins/editors/creators, as long as the discussion around rejecting/removing an app from the list on this ground happen in the open.

**Comment by csarven** [comment]
@Mitzi-Laszlo All of those questions could be answered depending on what you (authors) would like to publish at the target webpage. From an editorial perspective, there is information gap - I'm just suggesting what they are. Again, if the criteria is intended to be general rules, indicate that - it doesn't have to link to a particular "ethical framework", section, paragraph, ... but if you really want to, pass it through solid/process perhaps.

**Comment by csarven** [comment]
@NSeydoux That works really well, thanks!

File: pages/use-solid/apps.md [hunk 3]
```diff
@@ -6,7 +6,25 @@ permalink: /use-solid/apps
...
+
+The criteria for an app to be Solid-compatible are: 
+* If identifying users is necessary, they must be able to login using their WebID and pointing to the Identity Provider of their choice
+* Data consumed by the app should be fetched from Solid Pods if possible
+* Data generated by the app should be stored in Solid Pods
+* The interaction between the app, pods, and Identity Provider(s) must be compliant with the Solid specification
+
+Please note that apps falling into the following categories will not be listed on this page, even if they match the criteria above:
+* Apps intentionally or indirectly causing harm to anyone
+* Apps which purpose would be widely considered unethical
```
Comment (RubenVerborgh) [inline pages/use-solid/apps.md:19]: grammar seems off

File: pages/use-solid/apps.md [hunk 4]
```diff
@@ -6,7 +6,25 @@ permalink: /use-solid/apps
...
+* Apps designed to enable the generation and/or dissemination of hate speech
+* Apps encouraging stereotypes or violent behaviours towards any person or group
+* Apps having malicious intent, such as data theft
+* Apps designed to fullfill illegal purposes, such as drug traffic, stolen information exchange, extorsion...
+* Apps sharing users' information with any thrid-party without user consent
```
Comment (RubenVerborgh) [inline pages/use-solid/apps.md:24]:
```suggestion
* Apps sharing users' information with any third-party without user consent
```

**Comment by NSeydoux** [comment]
Since there has been editors approval earlier that was dismissed automatically when I merged in a typo fix by Ruben (?), we can go ahead and merge.

**Pull request merged**